File: database/port/EquipeJoueur.java
package database.port;

//[using]

@Table(name = "equipe_joueur")
class EquipeJoueur
{

    @Column(name = "id_equipe")
    public String idEquipe;    

    @Column(name = "id_saison")
    public String idSaison;    

    @Column(name = "id_equipe_joueur")
    public String idEquipeJoueur;    

    @Column(name = "dossart")
    public String dossart;    

    @Column(name = "id_joueur")
    public String idJoueur;    



    public void setIdEquipe(String value){
        this.idEquipe = value;
    }
    public String getIdEquipe(){
        return this.idEquipe;
    }

    public void setIdSaison(String value){
        this.idSaison = value;
    }
    public String getIdSaison(){
        return this.idSaison;
    }

    public void setIdEquipeJoueur(String value){
        this.idEquipeJoueur = value;
    }
    public String getIdEquipeJoueur(){
        return this.idEquipeJoueur;
    }

    public void setDossart(String value){
        this.dossart = value;
    }
    public String getDossart(){
        return this.dossart;
    }

    public void setIdJoueur(String value){
        this.idJoueur = value;
    }
    public String getIdJoueur(){
        return this.idJoueur;
    }

   
}

File: database/port/EquipeJoueur.cs
namespace database.port;

//[using]

[Table(name = "equipe_joueur")]
class EquipeJoueur
{

    [Column(name = "id_equipe")]
    public String idEquipe { get; set; }    

    [Column(name = "id_saison")]
    public String idSaison { get; set; }    

    [Column(name = "id_equipe_joueur")]
    public String idEquipeJoueur { get; set; }    

    [Column(name = "dossart")]
    public String dossart { get; set; }    

    [Column(name = "id_joueur")]
    public String idJoueur { get; set; }    



    public void SetIdEquipe(String value){
        this.idEquipe = value;
    }
    public String GetIdEquipe(){
        return this.idEquipe;
    }

    public void SetIdSaison(String value){
        this.idSaison = value;
    }
    public String GetIdSaison(){
        return this.idSaison;
    }

    public void SetIdEquipeJoueur(String value){
        this.idEquipeJoueur = value;
    }
    public String GetIdEquipeJoueur(){
        return this.idEquipeJoueur;
    }

    public void SetDossart(String value){
        this.dossart = value;
    }
    public String GetDossart(){
        return this.dossart;
    }

    public void SetIdJoueur(String value){
        this.idJoueur = value;
    }
    public String GetIdJoueur(){
        return this.idJoueur;
    }

   
}
